routing_model/index.html

```diff
@@ -132,8 +141,58 @@ <h2>Node as a member of a section</h2>
+  There are some subtleties, such as the fact that we only want to process one candidate at a time, but this is the general idea.
+  </p>
+  <button class="collapsible">RPC: ExpectCandidate</button>
+  <div class="content">
+    <p>We receive this RPC from a node that wants to join our section.<br>
+    They aren't connected to us, yet; so the RPC reaches us through their proxy node.<br>
+    On receiving it, we vote for ParsecExpectCandidate to process it in the same order as other members of our section.<br>
+    It kickstarts the entire chain of events in this diagram.
+    </p>
+  </div>
+  <button class="collapsible">RPC: ResourceProofResponse/<br>CandidateInfo</button>
+  <div class="content">
+  <p>These RPCs should generally be handled during resource proof, in AcceptAsCandidate.<br>
+  It could be that they reach us before we are ready to receive them. For instance, because although enough other nodes got consensus on ParsecExpectCandidate, I'm still a bit behind.<br>
+  It's OK to discard the RPC in this case as the node who aims to be rellocated here will perceive this as a timeout from our end and resend later.
+  </p>
+  </div>
+  <button class="collapsible">ParsecOnline<br>ParsecPurgeCandidate</button>
+  <div class=content>
+    <p>These RPCs come from AcceptAsCandidate.<br>
+    The same node could be accepted by some nodes who would vote ParsecOnline, but also time out for some other nodes who would vote for ParsecPurgeCandidate.<br>
+    If it's the case, we only want to process the first of these two events and discard the other one.<br>
+    Because the "Wait for 2" loop in AcceptAsCandidate has higher precendence than the "Wait for 1" loop in this one, this is exactly what will happen.
+    </p>
+  </div>
+  <button class="collapsible">shorter_prefix_section</button>
+  <div class="content">
+    <p>If we know of a section that has a shorter prefix than ours, we prefer for them to receive this incoming node rather than ourselves as it will help kepp the Network's sections tree balanced.<br>
+    This shorter_prefix_section is a function that will return None if we are the shortest of any section we know, Some if there is a better candidate.<br>
+    If it is Some, we will relay the RPC to them instead of ourselves.
+    </p>
+  </div>
+  <button class="collapsible">PROCESSING_CANDIDATE</button>
+  <div class="content">
+    <p>We want to allow at most one node at a time to enter our section through rellocation.<br>
+    The local variable: PROCESSING_CANDIDATE allows us to keep track of the fact someone is already passing the resource proof process, so we can't accept anyone else for now.<br>
+    When PROCESSING_CANDIDATE is true and we reach consensus on ParsecExpectCandidate, we simply send a RefuseCandidate RPC to the would-be-incoming-node so they know they must try another section or try again later.
```

Comment:
somewhat inaccurate:
- only refuse if we are the shortest_prefix.
- send it to the originating section, not the node.

> we simply

=> we
> so they know they must try another section or try again later.

=> so they can try another section or try again later